.github/workflows/codeql-analysis.yml

```
@@ -39,6 +39,11 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v2
+    - name: Set up (force) Go 1.17
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.17.2
```

Minor, but they're up to 1.17.7 - maybe wildcard the patch?
```suggestion
        go-version: 1.17.x
```

---

Yeah, agreed. I'll add suggestions in a new PR on top of develop, shouldn't block this.